### PR TITLE
Add Tests for storage for TAB

### PR DESCRIPTION
**Pull request by tanjoewel**
Changed/Added the following test files:

LogicManagerTest
JsonAdaptedModuleTest
JsonAdaptedStudentTest
JsonAdaptedTaskTest
JsonSerializableAssistantBuddyTest
JsonTeachingAssistantBuddyStorageTest
StorageManagerTest

### CODE REVIEW

**Review by WangGLJoseph — APPROVED** [review]
LGTM！

File: src/test/java/seedu/address/logic/LogicManagerTest.java [hunk 1]
```diff
@@ -1,162 +1,152 @@
...
+
+    @Test
+    public void execute_validCommand_success() throws Exception {
+        String listCommand = HomeCommand.COMMAND_WORD;
+        assertCommandSuccess(listCommand, HomeCommand.MESSAGE_SUCCESS, model);
+    }
+
+    @Test
+    public void execute_storageThrowsIoException_throwsCommandException() {
+        // Setup LogicManager with JsonAddressBookIoExceptionThrowingStub
```
Comment (felissaf) [inline src/test/java/seedu/address/logic/LogicManagerTest.java:63]: JsonTAB?

**Review by felissaf — COMMENTED** [review]
LGTM

File: src/test/java/seedu/address/logic/LogicManagerTest.java (hunk 1 above)
Comment (tanjoewel) [reply src/test/java/seedu/address/logic/LogicManagerTest.java:63]: Ah good catch! Ill fix this and then merge the PR!

**Pull request merged**